#!/usr/bin/env python
PACKAGE = "pcl_filters"
NODE = "downsampling_filter"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("voxel_width", double_t, 0, "voxel width(x value).", 0.01, 0, 0.1)
gen.add("voxel_length", double_t, 0, "voxel length(y value).", 0.01, 0, 0.1)
gen.add("voxel_height", double_t, 0, "voxel height(z value).", 0.01, 0, 0.1)

exit(gen.generate(PACKAGE, NODE, "downsampling"))
